pkgs/applications/networking/browsers/chromium/plugins.nix

```diff
@@ -98,12 +98,12 @@ let
 
   flash = stdenv.mkDerivation rec {
     name = "flashplayer-ppapi-${version}";
-    version = "29.0.0.171";
+    version = "30.0.0.113";
 
     src = fetchzip {
       url = "https://fpdownload.adobe.com/pub/flashplayer/pdc/"
           + "${version}/flash_player_ppapi_linux.x86_64.tar.gz";
-      sha256 = "1j7w81wjfrpkir11m719jdahnbnw4sph448hs90hvai6rsn3imib";
+      sha256 = "092fjp5znnc8mz6jh4nm9kswkhfr31v2l7kasr97f32rw9fksi9g";
```

Comment:
fixed-output derivation produced path '/nix/store/383lpd21lr1jjhk9q5dz1aicjhm9iy0p-source' with sha256 hash '0bcsrsz2dd12xs9vn2977k4s6hag1cknkrsgxz3c9pxk4jz99f3k' instead of the expected hash '092fjp5znnc8mz6jh4nm9kswkhfr31v2l7kasr97f32rw9fksi9g'

Reply:
Oops